caps/misc_test.go

```diff
@@ -42,8 +40,9 @@ func (s *MiscSuite) TestLoadBuiltInTypes(c *C) {
 	repo := NewRepository()
 	err := LoadBuiltInTypes(repo)
 	c.Assert(err, IsNil)
-	c.Assert(repo.types, testutil.Contains, BoolFileType)
-	c.Assert(repo.types, HasLen, 1) // Update this whenever new built-in type is added
+	c.Assert(repo.types, DeepEquals, []Type{
```

Comment:
Thanks, I agree about the internal-vs-contract point. I've tweaked the test to check what is really relevant -- that bool-file type is registered.